distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -23,6 +23,8 @@
 #
 
 #START DDF SETTINGS
+start.ddf=true
+start.solr=true
```

Comment:
Took these two properties from @rzwiefel 's work in progress. 
The directive was to make the components separable for starting/stopping